nbs/core.ipynb

```diff
@@ -292,7 +292,6 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #572.            splits = ufp.backtest_splits(
</pre><p>I think it will raise error if refit=True and n_windows=None and user only specifies test_size. We might want to move the conversion n_windows&lt;&gt;test_size in _no_refit_cross_validation (line 422 approx) to get n_windows from test_size.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
addressed in 96e87d9